evidence/pool.go

```diff
@@ -339,6 +340,15 @@ func (ei info) ToProto() (*evproto.Info, error) {
 		return nil, err
 	}
 
+	bytes := ei.ByteSize
```

Comment:
can't we just always use `evpb.Size()`?

Reply:
But this is the size in protobuf format. Is it not a different size when you marshal it into pure bytes form. I was using something like this before but I was getting different sizes on one of the tests